.github/workflows/openapi-compatibility-tests.yaml

```diff
@@ -0,0 +1,40 @@
+name: lakeFS OpenAPI Compatibility Tests
+
+on:
+  pull_request:
+    paths:
+      - "api/**"
+  push:
+    branches:
+      - master
+  workflow_dispatch: {}
+
+# These permissions are needed to interact with GitHub's OIDC Token endpoint.
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  swagger-backward-compatibility:
+    name: Run Swagger compatibility tests
```

Comment:
Can remove the "Run" from all the description - we run everything

Reply:
🧠 🐴